libretroshare/src/retroshare/rsnotify.h

```diff
@@ -206,44 +206,42 @@ class RsNotify
 		virtual bool setDisableAskPassword (const bool /*bValue*/) { return false ; }
 };
 
-class NotifyClient
+struct NotifyClient
```

Comment:
Because this struct as some methods you might want to keep it as a class. As we discussed already, we should keep "struct" for pure data structures without functionality.

Reply:
Done! Thanks!